api/storage_service.cc

```diff
@@ -1260,6 +1260,13 @@ void set_storage_service(http_context& ctx, routes& r, sharded<service::storage_
     });
 }
 
+enum class scrub_return {
+    successful = 0,
+    aborted,
+    unable_to_cancel,
```

Comment:
What does unable to cancel mean?

Reply:
I'm not sure I fully understand what they meant by it in C*.  Maybe if there was an attempt to stop the compaction but it finished anyway.

We don't have to ever return it.

Reply:
No, it can be present, but let's add a comment that it is not used in scylla.